### ISSUE TITLE
To-dos

### ISSUE THREAD

**Issue by wbinzhe** [wbinzhe/Climate_Retail#1]
1. Climate & Retail Literature
2. Proposal (Neilson and RCA application)

**Comment by wbinzhe** [comment]
@shoonlee how will we process the applications? would you mind I use the current version of the proposal to apply Neilson? I think it only requires a general framework of the project.

**Comment by shoonlee** [comment]
@wbinzhe As we're going to have a meeting anyway in a few days, let's have a brief discussion before sending it to Nielsen. Again, what Nielsen requires for the application are the most important aspects of any given project, so we'll have to talk about them iteratively. Let's use the Nielsen application as an opportunity to getting our head straight on these important questions. 

Before the meeting, could you please slightly update the document? 

  - Add more papers related to the retail sector responses to external shocks (Start with Handbury and Moshary in Zotero shared folder)
  - Fix the expected outcomes/contributions: I know the current version has a section called "expected outcomes", but I don't think it's what they're asking. Take a look at the attached file. 
[research_proposal_template.docx](https://github.com/wbinzhe/Climate_Retail/files/6677525/research_proposal_template.docx)

Let me know if you have any questions!

**Comment by wbinzhe** [comment]
> @wbinzhe As we're going to have a meeting anyway in a few days, let's have a brief discussion before sending it to Nielsen. Again, what Nielsen requires for the application are the most important aspects of any given project, so we'll have to talk about them iteratively. Let's use the Nielsen application as an opportunity to getting our head straight on these important questions.
> 
> Before the meeting, could you please slightly update the document?
> 
> * Add more papers related to the retail sector responses to external shocks (Start with Handbury and Moshary in Zotero shared folder)
> * Fix the expected outcomes/contributions: I know the current version has a section called "expected outcomes", but I don't think it's what they're asking. Take a look at the attached file.
>   [research_proposal_template.docx](https://github.com/wbinzhe/Climate_Retail/files/6677525/research_proposal_template.docx)
> 
> Let me know if you have any questions!

@shoonlee Yes I will revise the proposal before our next meeting on Monday. 
- I haven't done the all literature reading yet. Will keep adding. 
- The format of "expected outcomes" follows what Siqi put in their proposal to RCA. I guess for contribution, we can integrate it into the introduction/contribution subsection (i know currently the part is weak but I will work on it with more literature digested).

**Comment by shoonlee** [comment]
@wbinzhe Yea, I know. I think that part from Siqi's proposal isn't quite right.

**Comment by wbinzhe** [comment]
> 1. Climate & Retail Literature
> 2. Proposal (Neilson and RCA application)

Meeting with Siqi 06-28
- slides for update (with picture on dataset temperature) 
- Neilson application

Real estate story (talk to people?)

**Comment by shoonlee** [comment]
@wbinzhe Thanks for the Nielsen application & approval. While we set up the computer, did you have a chance to explore how each grid data is constructed? 
Specifically,
  - How many weather stations have been used to produce the 1 by 1 grid temperature data? If it is indeed true that the data has been interpolated, can you explore how many of the grids will have missing value without interpolation?
  - How 4 by 4 grid data is constructed (do we have unique observation at every 4 by 4 grid or is this interpolated as well)?

**Comment by wbinzhe** [comment]
2021-07-23
Package ‘exactextractr’: exact spatial algorithm
